File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        check = {}

        for index, num in enumerate(nums):
            com = target - num
            if com in check:
                return sorted([nums.index(com), index])
            check[num] = index
        

File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        int [] ans = new int[nums.length*2];

        
        for (int j = 0; j < nums.length; j++) {
            ans[j] = nums[j];
            ans[j + nums.length] = nums[j];
        }

        return ans;
    }
}